packages/server/src/routes/cron/notifyOnUnminted.ts

```diff
@@ -1,29 +1,134 @@
-import { Firestore, CollectionReference } from "@google-cloud/firestore";
-import { createApiRoute } from "..";
+import {
+  Firestore,
+  CollectionReference,
+  DocumentSnapshot
+} from "@google-cloud/firestore";
+import { auth as adminAuth, messaging as adminMessaging } from "firebase-admin";
+import { PhoneNumberUtil } from "google-libphonenumber";
+import * as moment from "moment";
+
 import { CronNotifyOnUnmintedApiEndpoint } from "@raha/api-shared/dist/routes/cron/definitions";
+
+import { createApiRoute } from "..";
 import { sendPushNotification } from "../../helpers/sendPushNotification";
-import { messaging as adminMessaging } from "firebase-admin";
+import {
+  calculateMaxMintableForMember,
+  MINT_CAP
+} from "../../helpers/calculateMaxMintableForMember";
+
+// Notify people if it is 12 o'clock in their local time.
+const HOUR_TO_NOTIFY = 12;
```

Comment:
https://momentjs.com/docs/#/get-set/hour/ I think hour is 0 indexed

Reply:
12 o'clock is already zero indexed, since midnight is 0-o'clock. the one weird place where we zero-index in everyday life lol